SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h

```diff
@@ -169,7 +124,9 @@ struct FSpatialLaunchConfigDescription
 		: bUseDefaultTemplateForRuntimeVariant(true)
 		, Template()
 		, World()
+		, MaxConcurrentWorkers(1000)
```

Comment:
ah do we now need to specify this?

Reply:
Yup. It's a new feature in the new runtime